### PR TITLE
Make multi-file mesh names unique

### PR DESCRIPTION
**Pull request by Simon-Steinmann**
Adds the main proto file name in front of the mesh names

### CODE REVIEW

File: urdf2webots/parserURDF.py [hunk 1]
```diff
@@ -850,7 +850,9 @@ def getLink(node, path):
         getVisual(link, node, path)
     if hasElement(node, 'collision'):
         getCollision(link, node, path)
-    return link
+    if not any([hasElement(node, 'inertial'), hasElement(node, 'visual'), hasElement(node, 'inecollisionrtial')]):
```
Comment (DavidMansolino) [inline urdf2webots/parserURDF.py:853]:
```suggestion
    if not any([hasElement(node, 'inertial'), hasElement(node, 'visual'), hasElement(node, 'collision')]):
```

**Comment by Simon-Steinmann** [comment]
Did you remove the effort limit fix on purpose?

**Comment by DavidMansolino** [comment]
> Did you remove the effort limit fix on purpose?

Yes, I found a simpler solution just by changing the default value here: https://github.com/cyberbotics/urdf2webots/pull/70/files#diff-725bae3adc35161533858377f44cdbfaR196

Let me know if this cause some issue in your test cases.

**Comment by Simon-Steinmann** [comment]
Did you do a different implementation of the torque fix, or did it just get lost somewhere?

**Comment by Simon-Steinmann** [comment]
ahh gotcha

**Comment by DavidMansolino** [comment]
> Did you do a different implementation of the torque fix, or did it just get lost somewhere?

Yes just changing the default value here https://github.com/cyberbotics/urdf2webots/pull/70/files#diff-725bae3adc35161533858377f44cdbfaR196 fixes the issue since if the `effort` is not defined it will use this default value.

**Pull request merged**